weddineo-core/pom.xml

```diff
@@ -3,19 +3,25 @@
 	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
 	<modelVersion>4.0.0</modelVersion>
 	<parent>
-		<groupId>org.springframework.boot</groupId>
-		<artifactId>spring-boot-starter-parent</artifactId>
-		<version>2.3.4.RELEASE</version>
-		<relativePath/> <!-- lookup parent from repository -->
+		<groupId>com.worm.code</groupId>
+		<artifactId>weddineo</artifactId>
+		<version>0.0.1-SNAPSHOT</version>
 	</parent>
-	<groupId>com.worm.code</groupId>
-	<artifactId>weddineo</artifactId>
+
+
+	<artifactId>weddineo-core</artifactId>
 	<version>0.0.1-SNAPSHOT</version>
 	<packaging>jar</packaging>
-	<name>weddineo</name>
+	<name>weddineo-core</name>
 	<description>Project for wedding industry</description>
 
     <properties>
+		<springboot.starter.parent.version>2.3.4.RELEASE</springboot.starter.parent.version>
```

Comment:
One variable for spring boot version will be ok. All starters should be in the same version.

Reply:
Ok. You right.